report_templates/host_-_system_currency.erb

```diff
@@ -0,0 +1,73 @@
+<%#
+name: Host - System currency
```

Comment:
When I read this I expected a report on the values of `LC_CURRENCY` but turns out that's `LC_MONETARY`. Perhaps:
```suggestion
name: Host - System errata currency
```